### DISCUSSION TITLE
Metasploit failed to install due a missing file

### DISCUSSION THREAD

**Discussion by edgardo-krause** [Homebrew/discussions#4363] [Casks]
### Output of `brew config`

```shell
nop
```

### Output of `brew doctor`

```shell
nop
```

### Description of issue

The file is missing:

==> Downloading https://osx.metasploit.com/metasploit-framework-6.3.9%2B20230318102817-1rapid7-1.x86_64.pkg
curl: (22) The requested URL returned error: 404                              

Error: Download failed on Cask 'metasploit' with message: Download failed: https://osx.metasploit.com/metasploit-framework-6.3.9%2B20230318102817-1rapid7-1.x86_64.pkg

![image](https://user-images.githubusercontent.com/116086204/228323805-c556a9ec-1719-4dbb-90e6-2a0bb12145b9.png)

### Relevant casks

metasploit

**Comment by p-linnane** [comment]
Fixed in https://github.com/Homebrew/homebrew-cask/pull/143906.
